msteams-platform/concepts/design/plan-responsive-tabs-for-teams-mobile.md

```diff
@@ -81,7 +81,7 @@ You can provide following access to users:
 * Insert data items to create visualizations.
 * Share visualizations in a channel or group chat.
 
-:::image type="content" source="../../assets/images/app-fundamentals/dashboarding-and-data-visualization-apps-enablement.png" alt-text="Enable users to create content such as charts graphics.":::
+:::image type="content" source="../../assets/images/app-fundamentals/dashboarding-and-data-visualization-apps-enablement.png" alt-text="The screenshot shows the steps to enable users to create content, such as charts and graphics.":::
```

Comment:
Are there steps for users to create charts and graphics in Teams?

Reply:
resolved